### PR TITLE
Add example

### PR DESCRIPTION
**Pull request by iskhakovt**
Add an example: top lessons by reaction as a part of an internship assignment.

### CODE REVIEW

File: examples/top_lessons_by_reactions.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+NUM_OF_REACTION_PAGES = 100
+NUM_OF_REACTION_TOP = 10
+
+lessons_method = 'lessons'
+recommendation_reactions_method = 'recommendation-reactions'
+
+
+def print_status(message, done, total):
+    print('{}: {}/{}'.format(message, done, total), end='\r' if done != total else '\n')
```
Comment (mehanig) [inline examples/top_lessons_by_reactions.py]: I like it! 👍

File: examples/top_lessons_by_reactions.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+for i in range(NUM_OF_REACTION_PAGES):
+    print_status('Loading', i+1, NUM_OF_REACTION_PAGES)
+    response = load_api(recommendation_reactions_method, page=i+1)
+
+    for reaction in response.get('recommendation-reactions'):
+        value = reaction.get('reaction')
+        course_reactons[reaction.get('lesson')] += value if value > 0 else -value - 1
+
+    if not response.get('meta').get('has_next'):
```
Comment (mehanig) [inline examples/top_lessons_by_reactions.py]: `meta` always exist, as well as `has_next` so no need to use get, can be just `response['meta']['has_next']`

**Comment by mehanig** [comment]
Looks good!

**Comment by iskhakovt** [comment]
Wait a bit.

**Comment by iskhakovt** [comment]
I had a bug and fixed it right now, shall I make an new pull request?

**Comment by mehanig** [comment]
Yes, please, and refer to the bug it fixes.

**Pull request merged**